pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -367,6 +367,10 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(ctx context.Context, cl
 					return err
 				}
 			case storagehelpers.GetPersistentVolumeClaimClass(claim) != "":
+				// The provisionClaim function may start a new asynchronous operation to provision a volume,
```

Comment:
I think the comments here should also explain why it is comparing against an empty string as that is handled differently compared to the case when the class is not set at all.

Reply:
@xing-yang added.